### DISCUSSION TITLE
Find the odd one

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#840] [Galaxy Trivia] [answered]
**Open at ten**
**Is it a rum**
**Grips Slice**
**See ace heck**

**Comment by peterbaccaro** [comment, marked as answer]
Open at ten = **PANETTONE**
Is it a rum = **TIRAMISU**
Grips Slice = **SPICE GIRLS**
See ace heck = **CHEESECAKE**

The odd one out is **Grips Slice** (i.e. SPICE GIRLS) as the others are all all popular desserts.
